packages/sdk/src/types/Loyalties.ts

```diff
@@ -0,0 +1,525 @@
+	object: 'list'
+	total: number
+	data_ref: string
+	vouchers?: {
+		id: string
+		code?: string
+		campaign?: string
+		campaign_id?: string
+		category?: string
+		type?: string
+		discount?: DiscountUnit | DiscountAmount | DiscountPercent
+		gift?: {
+			amount?: number
+			balance?: number
+		}
+		loyalty_card?: {
+			points?: number
+			balance?: number
+		}
+		start_date?: string
+		expiration_date?: string
+		validity_timeframe?: {
+			interval?: string
+			duration?: string
+		}
+		validity_day_of_week?: number[]
+		publish?: {
+			count?: number
+			entries?: string[]
+		}
+		redemption?: {
+			quantity?: number
+			redeemed_quantity?: number
+			redemption_entries?: string[]
+		}
+		active?: boolean
+		additional_info?: string
+		metadata?: Record<string, any>
+		is_referral_code?: boolean
+		holder_id?: string
+		updated_at?: string
+	}[]
+}
+
+export interface LoyaltiesCreateMember {
+	voucher?: string
+	channel?: string
+	customer?: {
+		id?: string
+		name?: string
+		email?: string
+		metadata?: Record<string, string>
+		description?: string
+		source_id?: string
+	}
+	metadata?: Record<string, string>
+}
+
+export interface LoyaltiesCreateMemberResponse {
+	id: string
+	code?: string
+	campaign?: string
+	campaign_id?: string
+	category?: string
+	type?: string
+	loyalty_card?: {
+		points?: number
+		balance?: number
+	}
+	start_date?: string
+	expiration_date?: string
+	validity_timeframe?: {
+		interval?: string
+		duration?: string
+	}
+	validity_day_of_week?: number[]
+	publish?: {
+		object?: string
+		count?: number
+		url?: string
+	}
+	redemption: {
+		object?: string
+		quantity?: number
+		redeemed_quantity?: number
+		url?: string
+	}
+	active?: boolean
+	additional_info?: string
+	metadata?: Record<string, any>
+	is_referral_code?: boolean
+	holder_id?: string
+	updated_at?: string
+}
+
+export type LoyaltiesGetMemberResponse = LoyaltiesCreateMemberResponse
+
+export interface LoyaltiesGetMemberActivitiesResponse {
+	object?: string
+	data_ref?: string
+	total?: number
+	activities?: {
+		id?: string
+		type?: string
+		object?: string
+		created_at?: string
+		data?: {
+			previous_attributes?: {
+				loyalty_card?: {
+					points?: number
+					balance?: number
+				}
+			}
+			object?: {
+				loyalty_card?: {
+					points?: number
+					balance?: number
+				}
+				object?: string
+			}
+		}
+	}[]
+}
+
+export interface LoyaltiesAddPoints {
+	points: number
+}
+
+export interface LoyaltiesAddPointsResponse {
+	points?: number
+	total?: number
+	balance?: number
+	type?: string
+	object?: string
+	related_object?: {
+		type?: string
+		id?: string
+	}
+}
+
+export interface LoyaltiesRedeemReward {
+	reward?: {
+		id?: string
+	}
+	metadata?: Record<string, any>
+}
+
+export interface LoyaltiesRedeemRewardResponse {
+	id: string
+	object?: string
+	date: string
+	customer_id?: string
+	tracking_id?: string
+	amount?: number
+	reward: {
+		id: string
+		assignment_id?: string
+		voucher: {
```

Comment:
No, most defiinietly it is not always present since you can have material reward.